NeoOnnx/src/Optimizer.cpp

```diff
@@ -0,0 +1,73 @@
+/* Copyright © 2017-2020 ABBYY Production LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--------------------------------------------------------------------------------------------------------------*/
+
+#include "Optimizer.h"
+
+namespace NeoOnnx {
+//--------------------------------------------------------------------------------------------------------------
```

Comment:
Don't overuse separators. They are not needed in this file.

Reply:
removed